backend/src/main/java/eliasyazdani/capstone/cupwithme/backend/player/Player.java

```diff
@@ -7,6 +7,7 @@
 public record Player(
         @Id
         String id,
+        String admin,
```

Comment:
String admin verstehe ich nicht ganz im Kontext der App. Hätte intuitiv Boolean erwartet aber später gibst du ja den username wieder als Wert für den Admin ein? 🤔

Reply:
Ich speichere den Benutzernamen als 'Admin' zusammen mit den Spielern und Turnieren ab, die von diesem Benutzer erstellt wurden, um sie später aufrufen und bearbeiten zu können. Ich habe die Bezeichnung 'Admin' gewählt, da ich in der Zukunft auch die Rolle 'Benutzer' hinzufügen möchte, um lediglich die Ergebnisse anzeigen zu können.

Reply:
Macht Sinn, dann würde ich es persönlich eher "role" nennen oder eben als boolean, je nachdem ob das nur der Admin für ein bestimmtes Spiel oder die gesamte Anwendung sein soll. Wenn ich das richtig verstanden habe.